src/js/media/flash.js

```diff
@@ -153,6 +153,10 @@ vjs.Flash.prototype['currentSrc'] = function(){
   }
 };
 
+vjs.Flash.prototype['currentSource'] = function(){
```

Comment:
Makes sense. I was trying to decipher the best way to inject it and these were the only places I saw access `currentSource_`. Didn't think to check that they were extending the `media.js` file (Was too focused on the Tech part).

Ty.